### PR TITLE
BUGZ 1029 : create bug fix

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.atlassian.net/browse/BUGZ-1029

resolves issues with create window open, not closing properly, entity list not popping up and entity properties not working

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20507/HighFidelity-Beta-PR15971-ddc678b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20507/HighFidelity-Quest-Beta-PR15971-ddc678b.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20507/HighFidelity-Beta-PR15971-ddc678b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20507/HighFidelity-Beta-Interface-PR15971-ddc678b.dmg

File: interface/src/ui/InteractiveWindow.cpp [hunk 1]
```diff
@@ -202,9 +217,9 @@ InteractiveWindow::InteractiveWindow(const QString& sourceUrl, const QVariantMap
     } else {
         auto offscreenUi = DependencyManager::get<OffscreenUi>();
         // Build the event bridge and wrapper on the main thread
-        offscreenUi->loadInNewContext(CONTENT_WINDOW_QML, [&](QQmlContext* context, QObject* object) {
+        offscreenUi->loadInNewContext(CONTENT_WINDOW_QML, [&, this](QQmlContext* context, QObject* object) {
```
Comment (danteruiz) [inline interface/src/ui/InteractiveWindow.cpp]: you do not need `this` in the lamda, `&` implicitly captures `this`. https://en.cppreference.com/w/cpp/language/lambda

Comment (amerhifi) [reply interface/src/ui/InteractiveWindow.cpp]: addressed, thank you

**Comment by amerhifi** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20524/HighFidelity-Beta-PR15971-f94b082.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20524/HighFidelity-Quest-Beta-PR15971-f94b082.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20528/HighFidelity-Beta-PR15971-f94b082.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20528/HighFidelity-Quest-Beta-PR15971-f94b082.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20528/HighFidelity-Beta-PR15971-f94b082.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20528/HighFidelity-Beta-Interface-PR15971-f94b082.dmg

**Comment by amerhifi** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20533/HighFidelity-Beta-PR15971-f94b082.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20533/HighFidelity-Quest-Beta-PR15971-f94b082.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20533/HighFidelity-Beta-PR15971-f94b082.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20533/HighFidelity-Beta-Interface-PR15971-f94b082.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20533/HighFidelity-Beta-PR15971-f94b082.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20533/HighFidelity-Beta-Interface-PR15971-f94b082.dmg

File: interface/src/ui/InteractiveWindow.h [hunk 2]
```diff
@@ -34,7 +34,23 @@ class QmlWindowProxy : public QmlWrapper {
...
+
+public slots:
 
+    void emitScriptEvent(const QVariant& scriptMessage);
+    void emitWebEvent(const QVariant& webMessage);
+
+signals:
+
+    void scriptEventReceived(const QVariant& message);
+    void webEventReceived(const QVariant& message);
```
Comment (huffman) [inline interface/src/ui/InteractiveWindow.h]: Should we remove `emitScriptEvent` and `webEventReceived`? I think they are both non-functional?

File: interface/src/ui/InteractiveWindow.cpp [hunk 3]
```diff
@@ -129,6 +137,12 @@ InteractiveWindow::InteractiveWindow(const QString& sourceUrl, const QVariantMap
         presentationMode = (InteractiveWindowPresentationMode) properties[PRESENTATION_MODE_PROPERTY].toInt();
     }
 
+    if (!_interactiveWindowProxy) {
```
Comment (huffman) [inline interface/src/ui/InteractiveWindow.cpp]: I think we can omit this `if` - it should always be null

File: interface/src/ui/InteractiveWindow.h [hunk 4]
```diff
@@ -309,6 +325,7 @@ protected slots:
 private:
     std::shared_ptr<QmlWindowProxy> _qmlWindowProxy;
     std::shared_ptr<DockWidget> _dockWidget { nullptr };
+    InteractiveWindowProxy *_interactiveWindowProxy{ nullptr };
```
Comment (huffman) [inline interface/src/ui/InteractiveWindow.h]: What do you think about using a `unique_ptr` here with a custom deleter to call `deleteLater()`?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20549/HighFidelity-Beta-PR15971-5d7286b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20549/HighFidelity-Beta-Interface-PR15971-5d7286b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20549/HighFidelity-Beta-PR15971-5d7286b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20549/HighFidelity-Quest-Beta-PR15971-5d7286b.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20548/HighFidelity-Beta-PR15971-db94585.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20548/HighFidelity-Beta-Interface-PR15971-db94585.exe

**Comment by shanzzam** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20566/HighFidelity-Beta-PR15971-5d7286b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20566/HighFidelity-Quest-Beta-PR15971-5d7286b.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20566/HighFidelity-Beta-PR15971-5d7286b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20566/HighFidelity-Beta-Interface-PR15971-5d7286b.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20566/HighFidelity-Beta-PR15971-5d7286b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20566/HighFidelity-Beta-Interface-PR15971-5d7286b.dmg

**Pull request closed without merging**